src/main/java/com/google/devtools/build/lib/syntax/MethodLibrary.java

```diff
@@ -170,14 +186,45 @@ private static boolean hasElementWithBooleanValue(
             type = Object.class,
             doc = "This collection.",
             // TODO(cparsons): This parameter should be positional-only.
-            legacyNamed = true)
+            legacyNamed = true),
+        @Param(
+            name = "key",
+            doc = "The key to sort with.",
+            named = true,
+            defaultValue = "None",
+            positional = false,
+            noneable = true),
+        @Param(
+            name = "reverse",
+            type = Boolean.class,
+            doc = "Whether to sort asc or desc.",
```

Comment:
Document boolean parameters using a verb phrase:
"return results in descending order"